cmd/wardend/Dockerfile

```
@@ -2,10 +1,38 @@
+    apt update && apt install -y \
+    libvips-dev \
+    ca-certificates curl
+
+RUN useradd -M -u 1000 -U -s /bin/sh -d /data warden && \
     install -o 1000 -g 1000 -d /data
-COPY ./wardend /usr/bin/wardend
+
+ARG WASMVM_VERSION
+ARG WASMVM_AMD64_CHECKSUM
+ARG WASMVM_ARM64_CHECKSUM
+ADD --checksum=sha256:${WASMVM_AMD64_CHECKSUM} --chmod=755 https://github.com/CosmWasm/wasmvm/releases/download/${WASMVM_VERSION}/libwasmvm.x86_64.so /usr/lib/libwasmvm.x86_64.so
```

Hadolint (2.12.0)

[error] invalid flag: --checksum

(DL1000)